README.md

```diff
@@ -322,6 +322,21 @@ default is 2GB, which is not enough to build p4c. Increase the memory limit to
 at least 4GB via Docker preferences or you are likely to see "internal compiler
 errors" from gcc which are caused by low memory.
 
+# Bazel
```

Comment:
This is cool.
One nice thing about the makefiles was that you could just add another project and if you respect the format of the makefiles it would build. For example, on my machine I checkout the p4c-xdp repo, and then I create a symlink in `extensions` to the root directory, and it is automatically picked up by cmake, including all tests. I don't really know bazel, but with this structure it doesn't look to me like this would behave in the same way.

Reply:
With Bazel this would work the opposite way, in the sense that your extension (p4c-xdp) would include p4c as a dependency. There is an example in the bazel/example folder of how to depend on p4c in a Bazel project.

Reply:
An extension works by adding a symlink from the extensions folder to a source directory. Then the *.def files and the Cmakefile are picked up by the build automatically. There are some conventions on the structure of the makefile. As an example you can see http://github.com/vmware/p4c-xdp. It would be nice to make sure that one can be added to the Bazel build without editing any of the files in this repository.